subprojects/deploy-cluster/entitlement.mk

```diff
@@ -0,0 +1,13 @@
+manifest_entitle:
+	@echo "PEM:	 ${ENTITLEMENT_PEM}"
+	@echo "RHSM:	 ${ENTITLEMENT_RHSM}"
+	@echo "Template: ${ENTITLEMENT_TEMPLATE}"
+	@echo "Dest:	 ${ENTITLEMENT_DST_BASENAME}*"
+	@cat ${ENTITLEMENT_TEMPLATE} \
+	  | sed "s/BASE64_ENCODED_PEM_FILE/$(shell base64 -w 0 ${ENTITLEMENT_PEM})/g" \
+	  | sed "s/BASE64_ENCODED_RHSM_FILE/$(shell base64 -w 0 ${ENTITLEMENT_RHSM})/g" \
+	  > ${ENTITLEMENT_DST_BASENAME}.yaml
+	@awk '{ print > "${ENTITLEMENT_DST_BASENAME}_"++i".yaml" }' RS='---\n' ${ENTITLEMENT_DST_BASENAME}.yaml
```

Comment:
not sure where I got that from, must be SO ;-)

I added a comment to explain what is done, just in case
```
	# Split "${ENTITLEMENT_DST_BASENAME}.yaml" into multiple files containing a single YAML object
	# openshift-install doesn't allow files with multiple objects
```